mail/postfix/src/opnsense/mvc/app/controllers/OPNsense/Postfix/forms/general.xml

```diff
@@ -42,6 +44,24 @@
         <help>The smtpd_banner parameter specifies the text that follows the 220 code in the SMTP server's greeting banner. Default is "'System Hostname' ESMTP Postfix".</help>
     </field>
     <field>
+        <id>general.message_size_limit</id>
+        <label>Message Size Limit</label>
+        <type>text</type>
+        <help>Set the max size for messages to accept, default is 501200000 Byte which is 50MB. Values must be entered in Bytes.</help>
+    </field>
+    <field>
+        <id>general.disable_ssl</id>
+        <label>Disable old SSL</label>
```

Comment:
I would suggest to rename that.

Reply:
To what? I first thought about Disable SSL, but this could lead to think disable encryption completely.

Reply:
TLS only, is this the default to do TLS only? If not, a good reason not to?

Reply:
With this approach sending and receiving would be tls only, if you have customers with really old equip only supporting SSL this could be disabled. We can also set the receiving part static to do only TLS since it's us and only change client connections to changeable.

Reply:
SSL should not be supported anywhere as it is broken. Only TLS should be used and therefore @mimugmail used the right setting: disable SSL by default.
Maybe "Allow Only TLS Transport Layer Encryption"

The suggestion of @fichtner may make people think that this disables plain text transmission.

Reply:
fixed